extensions/blocks/testimonial/save.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { RichText } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+//import { __ } from '../../utils/i18n';
+
+export default ( { attributes: { name, title, content, mediaUrl, mediaId }, className } ) => (
+	<div className={ className }>
+		<RichText.Content
+			tagName="div"
+			className="wp-block-jetpack-testimonial__content"
+			value={ content }
+		/>
+		{ mediaId && mediaUrl && <img src={ mediaUrl } alt="" /> }
```

Comment:
The `alt` attribute is currently blank. Is this intentional? It feels like it should have a valid value as the only time an empty `alt` is valid is when the image is always guaranteed to be _purely_ presentational in nature.

Reply:
I'm thinking about using the authors `name || title || ""` as the alt. Does it make sense?